migrations/20200719025533-add-name-column-users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
```

Comment:
is there a particular need to use this await/async syntax instead of the `Promise.all` syntax?
https://sequelize.org/master/manual/migrations.html#migration-skeleton